docs/notes/2.26.x.md

```diff
@@ -55,6 +55,21 @@ Pants has a new mechanism for `@rule` invocation in backends. In this release th
 - [visibility](https://www.pantsbuild.org/stable/reference/subsystems/visibility)
 - [yapf](https://www.pantsbuild.org/stable/reference/subsystems/yapf)
 
+#### Tool upgrades
+
```

Comment:
Ah, sorry, for the dribble of a review (I could've flagged earlier), but let's provide a bit more context:

```suggestion

The default versions of various external tools have been upgraded. The previous default versions are still supported, by pinning using the `version = "..."` field within the subsystem in `pants.toml`.

```

Reply:
Sure